### PR TITLE
chore: enable workflows delivery on demostration target

### PR DESCRIPTION
**Pull request by gregmagolan**
Tested here https://app.circleci.com/pipelines/github/aspect-build/aspect-cli/1226/workflows/31f4ed03-e2d2-46b9-9b17-cb0075c30c21

### CODE REVIEW

**Comment by kormide** [comment]
The deliver step appears to be failing in https://app.circleci.com/pipelines/github/aspect-build/aspect-cli/1221/workflows/f1b913e1-17d6-4dee-a0e7-a6b99b1279d2.

**Comment by gregmagolan** [comment]
> The deliver step appears to be failing in https://app.circleci.com/pipelines/github/aspect-build/aspect-cli/1221/workflows/f1b913e1-17d6-4dee-a0e7-a6b99b1279d2.

Yeah, looks like when I force pushed a new commit to the branch and the commit it was going to deliver was gone. Will kick it off again.

File: .aspect/workflows/bazelrc [hunk 1]
```diff
@@ -1,3 +1,3 @@
 # build without the bytes
-common --remote_download_minimal
-common --nobuild_runfile_links
+# common --remote_download_minimal
+# common --nobuild_runfile_links
```
Comment (kormide) [inline .aspect/workflows/bazelrc:3]: Intentional?

Comment (gregmagolan) [reply .aspect/workflows/bazelrc:3]: Temporary to green it up. Planning to deal with "build without the bytes" + delivery in a follow-up.

**Review by kormide — APPROVED** [review]

**Pull request merged**